module/epochs/epoch_lookup_test.go

```diff
@@ -170,44 +160,49 @@ func (suite *EpochLookupSuite) TestEpochForViewWithFallback_CurrNextPrev() {
 // EpochLookup with an initial state of epoch fallback triggered.
 func (suite *EpochLookupSuite) TestEpochForViewWithFallback_EpochFallbackTriggered() {
```

Comment:
I'm pretty sure this test is now identical to `TestEpochForViewWithFallback_CurrNextPrev`. If so, let's remove it.

Reply:
https://github.com/onflow/flow-go/pull/6247/commits/56b0e6236a1b8f3b6d2d7f902bb9acb0042b62c5